docs/index.rst

```diff
@@ -114,6 +114,14 @@
 .. toctree::
   :maxdepth: 0
   :hidden:
+  :caption: Stackdriver Error Reporting
```

Comment:
Full name is "Google Stackdriver Error Reporting". 
It is OK to remove "Google" is the Google context is obvious.
It is OK to remove "Stackdriver" is the Google Stackdriver context is obvious.

In this case, "Stackdriver Error Reporting" LGTM